Content.Server/Radio/EntitySystems/RadioSystem.cs

```diff
@@ -15,6 +15,11 @@
 using Robust.Shared.Utility;
 // ES START
 using Content.Server._ES.Radio;
+using Content.Server.Animals.Systems;
+using Content.Shared._ES.Degradation;
+using Content.Shared._ES.Masks.Traitor;
+using Content.Shared.Animals.Components;
+
```

Comment:
```suggestion
using Content.Shared.Animals.Components;
```
ominous space of doom